src/server/game/Reputation/ReputationMgr.cpp

```diff
@@ -41,6 +41,18 @@ ReputationRank ReputationMgr::ReputationToRank(int32 standing)
     return MIN_REPUTATION_RANK;
 }
 
+int32 ReputationMgr::ReputationRankToStanding(ReputationRank rank)
+{
+    int32 standing = Reputation_Bottom;
+    for (uint8 i = MIN_REPUTATION_RANK; i <= rank; ++i)
+    {
+        standing += PointsInRank[i];
+    }
+
+    return std::max(standing - 1, Reputation_Bottom);
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/16887899/149985654-88f76455-4216-4b5e-88bc-dd990dce861a.png)

Reply:
remove blank line at either line 54 or 55 to pass codestyle check

Reply:
@UltraNix

Reply:
codecheck broken build cant merge

Reply:
@UltraNix please check

> codecheck broken build cant merge

Reply:
Done.